apps/prairielearn/src/server.ts

```diff
@@ -2462,7 +2463,7 @@ if ((esMain(import.meta) || (isHMR && !isServerInitialized())) && config.startSe
     if (config.initNewsItems) {
       // We initialize news items asynchronously so that servers can boot up
       // in production as quickly as possible.
-      news_items.initInBackground({
+      await news_items.initInBackground({
```

Comment:
Shouldn't this function just be flagged as non-async?

Reply:
Yes, good catch. Done in ed32c0fd05e656102949ea224dc4336fbd3fba0f.